Creiwry/irl-meeting-app
# fix(back): modified navbar links to add messages view to correct icon…

… and make sure links available after sign in

### Ticket Link
(Found on Jira in your ticket description)
N/A

### User Story
(Linked to on Jira in your ticket description)
N/A

### What does this PR do?
(Your answer here.)
Fixes the navbar logic and links to add conversations_path to the correct icon button and remove it from the dropdown menu in the profile button. Also ensures that all links are available to the user after sign in. 

### Where should the reviewer start?
(Your answer here.)
navbar.html.erb was the only file modified.

### How should this be manually tested?
(Your answer here.)
Could you please test that after you have clicked on the conversations button you can still click on the profile button afterwards and see the options appear in the dropdown menu (edit profile and log out). Normally it works but I have the impression that the link becomes "dead" after I've gone back and forth a few times, which may just be a problem with rails server.

### Screenshots (if appropriate)
(Your answer here.)

### Are there any breaking changes?
No and all tests pass.
